endpoints/openrtb2/amp_auction.go

```diff
@@ -325,6 +340,7 @@ func sendAmpResponse(
 	labels metrics.Labels,
 	ao analytics.AmpObject,
 	errs []error,
+	seatNonBid *openrtb_ext.NonBidsWrapper,
```

Comment:
Why is this passed as a pointer / by reference?

Reply:
No need to pass it as pointer. corrected it.